Earthworks-12d-2/Earthworks-12d-2.ifc

```diff
@@ -38,7 +38,6 @@ DATA;
 #31 = IFCINDEXEDCOLOURMAP(#30, $, #32, (1, 1, 1, 1));
 #32 = IFCCOLOURRGBLIST(((5.88235294117647E-1, 3.52941176470588E-1, 0.)));
 #33 = IFCFACILITY('0lav5jLhj5Rvzt54vjJqYg', #2, 'a cut', 'Description of a cut', $, #15, $, $, .ELEMENT.);
-#34 = IFCRELCONTAINEDINSPATIALSTRUCTURE('0nG772dx13U8MkfDgKjZCi', #2, 'FacilityContainer', 'FacilityContainer for Elements', (#35), #33);
```

Comment:
Why deletion?

Reply:
general usage see [LINK](https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC2/HTML/schema/ifcproductextension/lexical/ifcfeatureelement.htm):

Spatial Containment
As a subordinate part being fully dependent on the master element the IfcFeatureElement shall have no independent containment relationship to the spatial structure.
The SELF\IfcElement.ContainedInStructure relationship shall be NIL.

therefore you cant have a `IfcRelContainedInSpatialStructure` for this only a `IfcRelVoidsElement` which it has.

Reply:
it was easier for me to merge and correct, instead of bothering lee as this was the only issue